torch/csrc/jit/runtime/static/impl.h

```diff
@@ -76,13 +77,25 @@ class TORCH_API StaticModule {
       const torch::jit::Module& m,
       const StaticModuleOptions& opts = StaticModuleOptions());
 
+ typedef enum {
+    CONSTANT_VALUE = -2, // VALUE nodes defined by prim::Constant
+    INPUT_VALUE = -1,    // VALUE nodes representing graph inputs
+    OTHER_VALUE = 0      // other VALUE nodes (use non-negative index)
+  } VALUE_KIND;
+
  private:
   explicit StaticModule(
       std::pair<
           std::shared_ptr<torch::jit::Graph>,
           c10::optional<c10::FunctionSchema>> graph_and_schema,
       const StaticModuleOptions& opts);
 
+  // for <kind, idx>
+  //   if kind == CONSTANT_KIND: map to constants_[idx]
+  //   if kind == INPUT_KIND: map to inputs_[idx]
+  //   otherwise: map to nodes_[kind].outputs()[idx]
+  typedef std::pair<VALUE_KIND, int> DEF_INFO;
```

Comment:
A minor naming convention thing, usually our typedefs are camel case, like `DefInfo`

Reply:
Good to know. Updated to `DefInfo`.